charts/lakefs/values.yaml

```diff
@@ -44,3 +44,8 @@ affinity: {}
 
 s3Fallback:
   enabled: false
+
+#committedLocalCacheVolume:
+# awsElasticBlockStore:
```

Comment:
not sure this is the best recommendation - ideally lakeFS should cache to an [NVMe instance store](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/InstanceStorage.html#instance-store-volumes) - no idea how to configure this in k8s though...